fearless/Modules/SoraCard/KYCMain/KYCMainPresenter.swift

```diff
@@ -66,6 +54,29 @@ extension KYCMainPresenter: KYCMainViewOutput {
             self?.router.present(viewModel: viewModel, from: self?.view)
         }
     }
+}
+
+// MARK: - KYCMainViewOutput
+
+extension KYCMainPresenter: KYCMainViewOutput {
+    func didTapIssueCardForFree() {
+        router.showTermsAndConditions(from: view)
+    }
+
+    func didTapGetMoreXor() {
+        if interactor.xorChainAssets.count > 1 {
```

Comment:
it's better to pass xorChainAssets to presenter when it's loaded, and use internal variable